packages/components/src/components/types/index.ts

```diff
@@ -0,0 +1,3 @@
+import { TChildren, TStyle } from './general.types';
```

Comment:
You can use `export * from './general.types';` instead. and remove the export line below.

Reply:
@hamid-deriv, this has proven problematic in the past. Have you tried it on your end?

Reply:
@jim-deriv We use this solution for exporting all the types in the cashier package and it just works.

Reply:
Sure thing, my first task in deriv was related to this though. Could you test it in this PR and see?

Reply:
Run `npm test` after

Reply:
@hamid-deriv, you may use this PR for your reference: https://github.com/binary-com/deriv-app/pull/6208

Reply:
@hamid-deriv, you may use this PR for your reference: https://github.com/binary-com/deriv-app/pull/6208